test/sparsematrix_ops.jl

```diff
@@ -4,7 +4,7 @@ module SparseTests
 
 using Test
 using SparseArrays
-using SparseArrays: getcolptr, nonzeroinds, _show_with_braille_patterns, _isnotzero
+using SparseArrays: getcolptr, nonzeros, nonzeroinds, _show_with_braille_patterns, _isnotzero
```

Comment:
Seems like `nonzeros` is already exported.

```suggestion
using SparseArrays: getcolptr, nonzeroinds, _show_with_braille_patterns, _isnotzero
```